deluge/ui/web/json_api.py

```diff
@@ -190,7 +191,9 @@ def _on_json_request(self, request):
         Handler to take the json data as a string and pass it on to the
         _handle_request method for further processing.
         """
-        content_type = request.getHeader(b'content-type').decode()
+        # We assume the all `application/json` content type will use
+        # the `utf-8` charset.
```

Comment:
I appreciate the addition of a comment but without any context about charset it is not worth mentioning here as comment. We should always err towards less comments unless the code we write is unclear but even then refactoring the code might help. 

The actual reasoning for making this change should be stored in the git commit message so we then use git blame in future if we are uncertain about this line of code. We also have the tests that could improved with a docstring to quickly outline reasoning.



I prefer that the test docstring mentions this and in the commit message provides more details about reasoning.